pages/use-solid.md

```diff
@@ -35,15 +35,22 @@ The following table lists some Pod Provider:
+| [trinpod.eu](https://trinpod.eu)                   | [Graphmetrix, Inc.](https://graphmetrix.com)                               | [Hetzner](https://www.hetzner.com/)           | Finland              | [TrinPod](https://graphmetrix.com/construction/trinpod)     |
+| [use.id](https://get.use.id/)                      | [Digita](https://www.digita.ai/)                                           | [DigitalOcean](https://digitalocean.com/)     | EU              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)     |
+| [solidweb.me](https://solidweb.me/)                | [Meisdata](https://github.com/serverproject-dev)                           | [Hosteurope](https://www.hosteurope.de)       | France              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
+| [Data Pod](https://datapod.igrant.io/)             | [iGrant.io, Sweden](https://igrant.io/)                                    | [RedPill Linpro](https://www.redpill-linpro.com/en), [AWS](https://aws.amazon.com/), [GCP](https://cloud.google.com/) or [Azure](https://azure.microsoft.com/en-in)        | EU              | [NSS](https://github.com/solid/node-solid-server)     |
+| [redpencil.io](https://solid.redpencil.io/)                 | [redpencil.io](https://redpencil.io/)                             | [Hetzner](https://www.hetzner.com/)           | Finland              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
+| [teamid.live](https://teamid.live/)                | [Meisdata](https://github.com/serverproject-dev)                           | [Hosteurope](https://www.hosteurope.de)       | France              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
+| [Open University UK](https://solid.open.ac.uk/  )  | [Open University UK](https://solid.open.ac.uk/local/pages/termsofuse.html) | [Jisc Services](https://www.jisc.ac.uk/)      | UK              | [NSS](https://github.com/solid/node-solid-server)           |
+| [KIT](https://solid.aifb.kit.edu/)                 | [Karlsruhe Institute of Technology](https://www.kit.edu/)                  | [Selfhosted](https://aifb.kit.edu)            | Germany              | [CSS](https://github.com/CommunitySolidServer/CommunitySolidServer)        |
+| [OpenLink Software(1)](https://solid.openlinksw.com:8444/) | [OpenLink Software](https://www.openlinksw.com/)                   | [Amazon](https://aws.amazon.com)              | USA              | [NSS](https://github.com/solid/node-solid-server)  |
+| [OpenLink Software(2)](https://solid.openlinksw.com:8445/) | [OpenLink Software](https://www.openlinksw.com/)                   | [Amazon](https://aws.amazon.com)              | USA              | [NSS](https://github.com/solid/node-solid-server)  |
```

Comment:
Seems like it is NSS with some OpenLink customisations? If that's a different software than NSS, it is better to link to that instead of NSS.

```suggestion
| [OpenLink Software(2)](https://solid.openlinksw.com:8445/) | [OpenLink Software](https://www.openlinksw.com/)                   | [Amazon](https://aws.amazon.com)              | USA              | OpenLink [NSS](https://github.com/solid/node-solid-server)  |
```